### ISSUE TITLE
Parallel Deployments of Stack Sets

### ISSUE THREAD

**Issue by maxrabin** [aws-solutions/quota-monitor-for-aws#97]
**Is your feature request related to a problem? Please describe.**

I am deploying the Query Monitor on an organization with many tens of accounts. It is deploying the Stack Sets Serially instead of in Parallel. It is very slooooowww.

**Describe the feature you'd like**

I would like the call to CreateStackSets to include Operation Preferences that request some level of parallelism, perhaps 10 at a time or 10% at a time.

**Additional context**

I believe the relevant code is [here](https://github.com/aws-solutions/quota-monitor-for-aws/blob/5e2770b38b120c76c51bcb2c7510c104e30788b8/source/lambda/utilsLayer/lib/cloudformation.ts#L70).

**Comment by maxrabin** [comment]
Another issue that would be solved with an improvement here is that the entire deployment of QM-TA-Spoke-StackSet got cancelled after a single deployment failed. There's 0 failure tolerance by default. Please add a reasonable failure tolerance, I suggest 10%.

The failure I got was `ResourceLogicalId:QMCWPollerQMCWPollerLambda824ABE36, ResourceType:AWS::Lambda::Function, ResourceStatusReason:Resource handler returned message: "The provided execution role does not have permissions to call SendMessage on SQS (Service: Lambda, Status Code: 400, Request ID:xxxxx-xxx-xx-xx-xxx)"`
